cmd.go

```diff
@@ -140,13 +141,20 @@ func init() {
 	txCmd.AddCommand(pushCmd)
 	txCmd.AddCommand(voteCmd)
 
-	txCmd.Flags().StringVarP(&flagTx, "tx", "t", "", "unsigned tx file")
-	txCmd.MarkFlagRequired("tx")
-	txCmd.Flags().IntVarP(&flagSequence, "sequence", "s", 0, "sequence number for the tx")
-	txCmd.Flags().IntVarP(&flagAccount, "account", "a", 0, "account number for the tx")
-	txCmd.Flags().StringVarP(&flagNode, "node", "n", "", "tendermint rpc node to get sequence and account number from")
-	txCmd.Flags().BoolVarP(&flagForce, "force", "f", false, "overwrite files already there")
-	txCmd.Flags().BoolVarP(&flagAdditional, "additional", "x", false, "add additional txs with higher sequence number")
+	pushCmd.Flags().StringVarP(&flagTx, "tx", "t", "", "unsigned tx file")
+	pushCmd.MarkFlagRequired("tx")
+	pushCmd.Flags().IntVarP(&flagSequence, "sequence", "s", 0, "sequence number for the tx")
+	pushCmd.Flags().IntVarP(&flagAccount, "account", "a", 0, "account number for the tx")
+	pushCmd.Flags().StringVarP(&flagNode, "node", "n", "", "tendermint rpc node to get sequence and account number from")
+	pushCmd.Flags().BoolVarP(&flagForce, "force", "f", false, "overwrite files already there")
+	pushCmd.Flags().BoolVarP(&flagAdditional, "additional", "x", false, "add additional txs with higher sequence number")
+
+	voteCmd.Flags().StringVarP(&flagDenom, "denom", "d", "", "fee denom, for offline creation")
+	voteCmd.Flags().IntVarP(&flagSequence, "sequence", "s", 0, "sequence number for the tx")
+	voteCmd.Flags().IntVarP(&flagAccount, "account", "a", 0, "account number for the tx")
+	voteCmd.Flags().StringVarP(&flagNode, "node", "n", "", "tendermint rpc node to get sequence and account number from")
+	voteCmd.Flags().BoolVarP(&flagForce, "force", "f", false, "overwrite files already there")
+	voteCmd.Flags().BoolVarP(&flagAdditional, "additional", "x", false, "add additional txs with higher sequence number")
```

Comment:
shouldnt these flags be on the general txCmd rather than the specific voteCmd? presumably we'll want them for all txs we generate

Reply:
The subcommands of `tx` didn't inherit the parameters from `tx` for some reason (unlike global flags) so the user experience was subpar because the `tx vote --help` command didn't list all the possible options. We're going to investigate in a new issue if there's some kind of binding or grouping available in `viper` so we don't copy-paste to each subcommand but for now this was a happy path that made it the simplest approach with the best UX.